test/subproc-specs.js

```diff
@@ -210,22 +216,18 @@ describe('SubProcess', function () {
 
   describe('#stop', function () {
     it('should send the right signal to stop a proc', async function () {
-      return await new B(async (resolve, reject) => {
-        let subproc = new SubProcess('tail', ['-f', path.resolve(__filename)]);
+        const subproc = new SubProcess('tail', ['-f', path.resolve(__filename)]);
+
         await subproc.start();
+
         subproc.on('exit', (code, signal) => {
-          try {
             signal.should.equal(stopSignal);
-            resolve();
```

Comment:
I am not sure this change works as expected in case of a failure

Reply:
in the old code it would reject a promise which would return an exception and fail the test. now it just throws directly and also stops the test, so there's virtually no difference:

```
 1) SubProcess
       #stop
         should send the right signal to stop a proc:
     Uncaught 
  Error
      at SubProcess.<anonymous> (test/subproc-specs.js:225:19)
      at SubProcess.emit (node:events:520:28)
      at SubProcess.emit (node:domain:488:12)
      at ChildProcess.<anonymous> (lib/subprocess.js:193:14)
      at ChildProcess.emit (node:events:520:28)
      at ChildProcess.emit (node:domain:488:12)
      at Process.ChildProcess._handle.onexit (node:internal/child_process:294:12)
```